2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
+---
+
+The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held live in Berlin from September 9th to 11th. Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.
```

Comment:
probably worth mentioning here that it will be live-streamed and accessible remotely

Reply:
and linking to the actual summit website with more info

Reply:
Done